src/Sastrawi/StopWordRemover/StopWordRemover.py

```diff
@@ -10,11 +10,9 @@ def get_dictionary(self):
     def remove(self, text):
         """Remove stop words."""
         words = text.split(' ')
-        for word in words:
-            if self.dictionary.contains(word):
-                words.remove(word)
+        stopped_words = [word for word in words if not self.dictionary.contains(word)]
```

Comment:
sedikit saran: variable `stopped_words` diganti karena isinya adalah text yg sudah tidak mengandung stop words lagi, kalau dinamai `stopped_words` khawatir rancu. How about `clean_words`? Thanks for the contribution anyway